fastlane/lib/fastlane/actions/appetize.rb

```diff
@@ -161,6 +172,7 @@ def self.example_code
         [
           'appetize(
             path: "./MyApp.zip",
+            api_host: "company.appetize.io", # only needed for enterprise hosted solution
```

Comment:
Can we move this to a second example that uses it so the first one stays the default case? Thanks.

Reply:
Done :-)